docs/source/backend/architecture.rst

```diff
@@ -0,0 +1,18 @@
+Architecture
+============
+
+The os2phonebook backend consists of two components:
+
+* A Python program (located in the ``import`` directory) that imports
+  all the necessary data from MO and stores it in the file system, and
+* An SOLR server in which the data are indexed, and from which the
+  frontend retrieves them.
+
+This corresponds to the two steps in the ``import_data.sh`` script.
+
+The phonebook is supposed to be kept up to date by importing and
+reindexing regularly, normally once a day. When reindexing, the SOLR
+data are completely removed and reimported, which means that data may be
```

Comment:
Data is singular — perhaps “the data in SOLR is”?

Reply:
That is [up for discussion](https://www.theguardian.com/news/datablog/2010/jul/16/data-plural-singular) among Anglophones. I'll leave it as is until they agree. :)